testutil/localstride/scripts/setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+CHAIN_ID=localstride
+STRIDE_HOME=$HOME/.stride
+CONFIG_FOLDER=$STRIDE_HOME/config
+MONIKER=val
+
+MNEMONIC="deer gaze swear marine one perfect hero twice turkey symbol mushroom hub escape accident prevent rifle horse arena secret endless panel equal rely payment"
+
+install_prerequisites () {
+    apk add dasel
+}
+
+edit_genesis () {
+
+    GENESIS=$CONFIG_FOLDER/genesis.json
+
+    # Update staking module
+    dasel put string -f $GENESIS '.app_state.staking.params.bond_denom' 'ustrd'
+    dasel put string -f $GENESIS '.app_state.staking.params.unbonding_time' '240s'
+
+    # Update crisis module
+    dasel put string -f $GENESIS '.app_state.crisis.constant_fee.denom' 'ustrd'
+
+    # Udpate gov module
+    dasel put string -f $GENESIS '.app_state.gov.voting_params.voting_period' '60s'
+    dasel put string -f $GENESIS '.app_state.gov.deposit_params.min_deposit.[0].denom' 'ustrd'
+
+    # Update epochs module
+    dasel put string -f $GENESIS '.app_state.epochs.epochs.[1].duration' "60s"
```

Comment:
Can we shorten both the "day" epoch and "stride_epoch" here? I believe they're index 0 and 2, but it's a little safer to just update via name instead of index. 

I'm not sure what the syntax is in dasel for it, but it's the following in jq so I assume it's similar:
```
'(.app_state.epochs.epochs[] | select(.identifier=="day")).duration = 120s' 
'(.app_state.epochs.epochs[] | select(.identifier=="stride_epoch")).duration = 120s'  
```